### PR TITLE
Rs 2022.10.17

### PR DESCRIPTION
**Pull request by rschlaikjer**
Sigma stare
2022-10-19-231729_1920x1140_scrot

### CODE REVIEW

**Comment by rschlaikjer** [comment]
This file has a bunch of わたし vs 私 vs 『私』. Do we want to try and encode the Roa/Ciel dichotomy in some way? All of Roa's lines in italics? Just Roa's first person pronouns in italics? Currently it's a bit of a mix of Is and 'I's.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 1]
```diff
@@ -474,95 +512,101 @@ Carefully. Carefully.
...
-Though 'I' may be the owner of this body, its thought processes and standards of thought have not entirely changed.
+Even though 'I' was the owner of this body,
+ its thought patterns and critical abilities hadn't changed at all.
 }
 [sha:b83b9c7f7bf172a85655fc1bac56bb05e27dcb58]{
 -- Page 534, Offset 19631.
 -- つまり―――わたしの肉体は『私』という転生者のものだったけれど、意識はもとのわたしのままだった。
 // this わたし　私 stuff might be a bit too convoluted in en
-In other words, though my body may belong to the reincarnated 'me', my consciousness is still mine.
+In other words, though my body may have belonged to the reincarnated 'me', my consciousness was still mine.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:566]: >Which is to say:...

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 2]
```diff
@@ -474,95 +512,101 @@ Carefully. Carefully.
...
+I could remember the sensation of biting into my parents' necks.
 }
 [sha:5bbd7ca40aaea0bca9665e7d6dbb8b132cb3cf0e]{
 -- Page 535, Offset 19636.
 -- わたしに笑顔を向けてくれた街の人たちひとりひとり、じっくりと足元から熔かすように、その魂ごと陵辱し続けた日々を。
 // that is one hell of a sentence
-I recalled each day I spent disgracing the very souls of everyone who directed their smiles at me, as if slowly melting them from the feet up.
+// Uhh doublecheck pls
+I could remember each day that I continued to defile the very souls of each and every person in the town who had once smiled at me,
+ as if I were slowly dissolving them all from the feet up.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:594]: >I could remember how, day after day, one by one, I would defile the souls people in the town who had smiled at me, slowly dissolving them from the feet up.

ように here isn't a comparison (disturbingly).

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 3]
```diff
@@ -572,82 +616,82 @@ Gone mad and abandoned everything to 'me'.
...
 -- Page 538, Offset 19649.
 -- その為に、より多くの街を、わたしの街と同じような死都に変えた<ヤ|・><ツ|・><ラ|・>が集まってきた。
-For that reason, %{i}they%{/i} gathered. They, who just like me, have turned many other towns into ghost towns.
+It's the reason that %{i}they%{/i}, each responsible for turning even more towns into ghost towns, had gathered there.
 }
 [sha:f4d2140b1a486797694a571536e2a7243dc73103]{
 -- Page 539, Offset 19650.
 -- わたしの意識はあと少しで燃え尽きる。
-Just a little more until my consciousness is burnt away.
+Not long now before my consciousness finally burns out.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:664]: > It wasn't long before my consciousness would burn out.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 4]
```diff
@@ -657,25 +701,27 @@ The sixth ceremony had begun.
...
 -- 『私』はわたしという肉体を得て、ようやくあの白い女を手に入れられると判断し、そして―――
-'I' obtained the body I once possessed, then judged that 'I' would finally be able to make the woman in white mine―――
+Having obtained the body that had once been mine,
+ 'I' thought that they'd finally be able to make the woman in white mine, and then―――
 }
 [sha:d62390bde028dbf1f7d9b7efc6f972c17dd5f913]{
 -- Page 542, Offset 19659.
 -- ……そして、あってはならない事変は収束した。
-...Then, the incident that should not have been came to an end.
+And then... the incident that should never have been came to an end.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:710]: rephrase to avoid repetition with 'end' below.

>And then... the incident that should never have been came to its conclusion.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 5]
```diff
@@ -657,25 +701,27 @@ The sixth ceremony had begun.
...
 }
 [sha:9bcff5ad614a407661ad60e49ca4532307a1dd1a]{
 -- Page 542, Offset 19660.
 -- 戦いの末、あの白い吸血姫は『私』を仕留めたのだ。
-At the end of the battle, that white vampire princess killed 'me'.
+At the end of the battle, the vampire princess in white killed 'me'.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:715]: > At the end of the battle, the vampire princess in white _had_ killed 'me'.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 6]
```diff
@@ -657,25 +701,27 @@ The sixth ceremony had begun.
...
 }
 [sha:481c251ba9a31c6010ae1afd23db9aba064d976e]{
 -- Page 542, Offset 19661.
 -- すでに転生の準備を進めていた『私』はまたも転生し、残されたものはわたしの遺体だけ。
-'I' reincarnated once again, having already made the preparations for it, leaving behind my corpse.
+'I' had already made preparations for their next reincarnation and was born again,
+ leaving behind nothing but my corpse.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:721]: pronoun weirdness again, though I suppose we'll have to do a general overview of how to deal with this.

(it returns later again)

**Review by Petrikow — COMMENTED** [review]
otu

day 8 truly is the start of the kinography

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 7]
```diff
@@ -1,173 +1,190 @@
...
 -- Page 504, Offset 19535.
 -- 何か、考え事があった気がするんだけど……」
-I feel like there was something weighing on my mind."
+I feel like I'm forgetting something..."
 }
 [sha:5cdc68d3a3bb8bb383c7b3d5b09c60ad4ac23987]{
 -- Page 505, Offset 19536.
 -- 今夜も眠気に負けてしまう。
 // i read this as morbius
+// Morpheus, Dorpheus, Orpheus / Go eat some walruses / Orifices, Porridges / Morpheus Morpheus
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:46]: Morpheus drinkin a 40 in a death basket

that said, are we sure we want to use this? i don't exactly think and as a result
1. i ain't exactly
2. I was like "huh? morpheus? what?" while the beautiful face of lawrence fishburne flashed before my eyes. 


Sure I eventually trudged up some thought to figure out what this meant, but for all the non-thinkers out there like myself, my vote is for
> Tonight, I'll surrender myself to sleep.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 8]
```diff
@@ -1,173 +1,190 @@
...
 -- Page 506, Offset 19539.
 -- 掛け布団を剥がして立ち上がる。
-I remove the blanket and get up from the bed.
+I kick off the blankets and get out of bed.
 }
 [sha:c88b84d39581f326f567320bbda67cc7c05be664]{
 -- Page 506, Offset 19540.
 -- なまじ体力が余っているので寝付けない。
-Unfortunately, it seems like I've got some leftover energy which is keeping me from falling asleep.
+My body's still too restless, I can't get to sleep.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:68]: kinda munges it a bit but
> It's like my body's still too restless; I can't get back to sleep.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 9]
```diff
@@ -1,173 +1,190 @@
...
 -- わたしはフランスの片田舎の、一商人の子として生を<享|う>けた。
-I was born as the child of a shopkeeper in the French countryside.
+I was born as the child of a shopkeeper deep in the French countryside.
 }
 [sha:8b7873ce2d024560bcabc7d84833cd0f5f5637cd]{
 -- Page 509, Offset 19547.
 -- わたしの容姿は東洋人の母ゆずりのもので、街のどこにいても異邦人のような違和感を持って育った。
-I inherited my mother's Far-Eastern features, so I'd always felt a strange sense of discomfort when I was in town, like I didn't belong.
+I inherited my mother's Far-Eastern features,
+ and so the entire time I was growing up I never quite felt like I belonged in the town I lived in.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:108]: _sigh_ fata morgana reference number 2

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 10]
```diff
@@ -1,173 +1,190 @@
...
+I helped my father, I went to school, and I dreamt of the future.
 }
 [sha:b15e0d2b64eb3f4c06fa269f40f96042ba94f51b]{
 -- Page 510, Offset 19550.
 -- 他の子に比べると早熟なコトにもちょっと悩んで。
-I was a bit worried about being more precocious than the other children.
+// Kinda confused by the actual meaning here. Feels kinda unusual for a child to be worried they themselves are
+// too precocious, but maybe if you're precocious enough as a child it gets that self referential.
+// Guessing the worry is 'why am I so smurt' and the answer is :roa:?
+I was also a little worried about how I was more precocious than the other children.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:127]: wait, she's 12 years old _physically_ or 12 years old _mentally_?

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 11]
```diff
@@ -1,173 +1,190 @@
...
 -- たとえばトモダチと遊んでいるとき。
-For instance, when I played with my %{i}friends%{/i}.
+// I don't think トモダチ is actually emphasis enough to be worth italics here?
+It would happen when I was playing with my friends.
 }
 [sha:40678056207d76359f8b3a8fdfe2739be560413f]{
 -- Page 511, Offset 19555.
 -- 理由もなく、そのか細い首をへし折りたくなる。
-For no particular reason, I felt the desire to break their slender necks.
+For no particular reason, I'd feel the desire to break their slender necks.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:154]: nah, I prefer break here

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 12]
```diff
@@ -222,47 +240,50 @@ I remained levelheaded until the end.
...
 -- 結論として。
-In summation:
+I came to a conclusion.
 }
 [sha:34bbb5c0342106538e0acf3b8faf865ea527701e]{
 -- Page 517, Offset 19576.
 -- わたしのこの嗜好は、誰にだってある、ほんのささいな破壊衝動なのだと受け入れた。
-I accepted that this inclination of mine was simply a destructive impulse that anyone could develop.
+// l'appel du ~vide~ violence
+I decided that these impulses of mine were just the same fleeting destructive impulses that everyone has sometimes.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:264]: maybe even "occasionally feels"

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 13]
```diff
@@ -282,57 +303,63 @@ I had no choice but to spend my days without seeing anyone, without doing anythi
...
 -- ……けれど、それは間違いだった。
-But I was wrong.
+Except... that proved to be a mistake.
 }
 [sha:eda0ec3b7208a84a1edb799c33e3231e34af9a98]{
 -- Page 521, Offset 19586.
 -- わたしはもっと神さまを信じて、自らの<性|さが>を恐れず、神父さまに相談するべきだったのだ。
-I should've had more faith in God. I should've sought counsel with a priest, without fearing my own nature.
+I should have had more faith in God.
+ I should have sought counsel with a priest, rather than hiding in fear from my own nature.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:317]: hmm, not sure i agree.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 14]
```diff
@@ -282,57 +303,63 @@ I had no choice but to spend my days without seeing anyone, without doing anythi
...
 -- 心配そうに語りかけてくる父と、涙を浮かべて微笑む母に出迎えられて、わたしはふたりを殺害した。
-My parents greeted me. My father sounded worried, while my mother smiled at me with tears in her eyes. I killed them both.
+My father spoke to me with care and worry in his voice, while my mother just smiled at me with tears in her eyes.
+ I killed them both.
 }
 [sha:2e9da86f89e8e0a89b9cdb5ee00c9c1408b0da5c]{
 -- Page 524, Offset 19593.
 -- 本当に、あっけなかった。
-It was, truly, disappointingly brief.
+It was over far too soon.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:357]: Are you sure? I don't think I really understand where you're getting that from since it reads more like lit. "Really, was unsatisfying" to me

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 15]
```diff
@@ -423,47 +455,53 @@ As a result, I had never come across a truly superior body.
...
 -- こと■回目の転生にして、私はようやく、その真実にたどり着いた。
-After reincarnating ■ times, I finally reached that truth.
+After reincarnating ■■ times, I finally understood that truth.
 }
 [sha:ea2d9bf95aa65d21c3dc460fb2c9dfe16902fe07]{
 -- Page 531, Offset 19619.
 -- 目覚めた私は、ごく自然に、水が地面を浸食していくように、ひそやかに街を死都に変えていった。
-Upon my awakening, as naturally as water erodes the ground, I secretly transformed this place into a ghost town.
+After I'd awoken, I surreptitiously transformed this hamlet into a ghost town,
+ as naturally as water slowly carves a path through the earth.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:497]: i concur, necropolis is a badass word

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 8/08_01_CIEL08_6.txt (hunk 10 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:127]: Physically.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt (hunk 11 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:154]: As far as I understand, break is more often used intransitively, while snap is more common in transitive cases (like this one).

File: script/Ciel/Day 8/08_01_CIEL08_6.txt (hunk 7 above)
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:46]: It kinda like it, but I suppose its one of those times when one ought to remember that this is a translation, and there is little to justify in the original text for such a sentence. Well, it's not exactly a big problem or anything. Personally speaking I found it fairly easily understood, but it doesn't justify itself very well.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:357]: It's more like an extension of meaning, but I can see it being a bit far-removed from the original in that way, I suppose.

あっけない = something which is so easy that it feels disappointing.
So what I felt was that, it was such a 'nothing-burger' to Ciel, that it basically feels like she's commenting on how little it stimulated her emotions = she didn't care = she didn't even think twice.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 16]
```diff
@@ -423,47 +455,53 @@ As a result, I had never come across a truly superior body.
...
 -- Page 530, Offset 19614.
 -- 私は歴代の私の計算高さを<嗤|わら>った。
-I laughed at the ridiculousness of the calculations made by my previous selves.
+I laughed at how amateurish the calculations made by my previous selves had been.
 }
 [sha:9d45ffd18fc7b44f098c91442b3d7f08d49f92dc]{
 -- Page 530, Offset 19615.
 -- 条件は逆にするべきだったのだ。
-I should have reversed the priority of the conditions.
+I had my priorities completely backwards.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:476]: >I'd had my priorities completely backwards

File: script/Ciel/Day 8/08_01_CIEL08_6.txt (hunk 10 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:127]: Gotcha, I assumed this was mental precociousness but it's more like developing early

File: script/Ciel/Day 8/08_01_CIEL08_6.txt (hunk 13 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:317]: Continuous feels more natural here for me too

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 17]
```diff
@@ -474,95 +512,101 @@ Carefully. Carefully.
...
 -- 真っ白いケーキの生地に、まずは絶望というクリームを一気に伸ばして、あとは少しずつ少しずつ
-On top of the pure white cake dough, first spread the cream of despair in one swoop, then little by little―――
+First, in one fell swoop, I spread a even layer of the cream of despair all over the pure white canvas of the cake.
+ Then, little by little―――
 }
 [sha:085d8814df6d01b7e54e2f9ef833cd97b06d971d]{
 -- Page 532, Offset 19624.
 -- あざ笑いながら、
-With a mocking smile plastered on my face―――
+With a mocking smile plastered across my face―――
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:528]: Eeh, I think that changes the pacing a bit much. I'd prefer to keep meat at the end.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 18]
```diff
@@ -572,82 +616,82 @@ Gone mad and abandoned everything to 'me'.
...
 -- Page 540, Offset 19654.
 -- その白い女は、赤い月と共にやってきた。
-That woman in white came together with the crimson moon.
+The woman in white arrived alongside the crimson moon.
 }
 [sha:88fdf23db34a0b74208cd8afcb54a86700f2da30]{
 -- Page 540, Offset 19655.
 -- わたしは知らない。
-I didn't know.
+I didn't recognize them.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:689]: Not sure what you mean, I went with them since it's more generic?

File: script/Ciel/Day 8/08_01_CIEL08_6.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_6.txt:718]: I think the pronouns are correct on this one? At least for the fucky kind of correct that this scene entails? 'Roa' had already made preparations ... and was born again, leaving behind nothing but Ciel's corpse.

File: script/Ciel/Day 8/08_01_CIEL08_6.txt [hunk 19]
```diff
@@ -282,57 +300,63 @@ I had no choice but to spend my days without seeing anyone, without doing anythi
 [sha:65b55fc279fa3e3dd4e7cd1f82da29b886958d45]{
 -- Page 520, Offset 19584.
 -- ……そうすれば、誰を恨むコトもなくて、何に<憤|いきどお>るコトもなくなる。
-...If I did that, there wouldn't be anything to get angry at, nor would there be anyone to resent.
+If I did that, there'd be nobody for me to get angry at. There'd be nothing for me to resent.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_6.txt:303]: > ...As long as I did that
